features/step_definitions/web_steps.rb

```diff
@@ -60,6 +60,18 @@ def with_scope(locator, &block)
   page.execute_script('$(tinymce.editors[0].setContent("' + value + '"))')
 end
 
+When(/^(?:|I )follow the first "([^"]*)" link$/) do |link_text|
```

Comment:
In the future, I would probably not add to web_steps, which is kind of generic and not very app-specific.